packages/rlc-common/src/interfaces.ts

```diff
@@ -79,6 +79,7 @@ export interface PathMetadata {
 export type Paths = Record<string, PathMetadata>;
 
 export type PathParameter = {
+  oriName?: string;
```

Comment:
What does this mean? Could we have a more meaningful name?

Reply:
originalName, this is because we have parameter named client in the client constructor, and in RLC client factory function client already been used. that's why we need to rename it as clientParam.  but client parameter also need to be used to build the base url. we need to know the original parameter name that we need to replace.
<img width="433" alt="image" src="https://github.com/Azure/autorest.typescript/assets/55688292/c9dbf43c-13d6-4adc-8a34-b5cef556ecc6">